test/commands/test_clean.js

```diff
@@ -25,18 +25,22 @@
 const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
+const os = require('os');
 const {runSync} = require('../helpers');
 
 describe('clean', function() {
   it('deletes all temporary files', function(done) {
-    home = path.resolve('temp/test-clean/simple');
+    const home = path.resolve('temp/test-clean/simple');
+    const eo = path.join(os.homedir(), '.eo');
     fs.rmSync(home, {recursive: true, force: true});
+    fs.rmSync(eo, {recursive: true, force: true});
     fs.mkdirSync(path.resolve(home, 'src'), {recursive: true});
+    fs.mkdirSync(eo, {recursive: true});
     fs.writeFileSync(path.resolve(home, 'src/simple.eo'), '[] > simple\n');
     const stdout = runSync([
-      'clean', '-s', path.resolve(home, 'src'), '-t', path.resolve(home, 'target'),
+      'clean', '-s', path.resolve(home, 'src'), '-t', path.resolve(home, 'target'), '--cached',
     ]);
-    assert(!fs.existsSync(path.resolve(home, 'target')), stdout);
+    assert(!fs.existsSync(path.resolve(home, 'target')) && !fs.existsSync(eo), stdout);
```

Comment:
@maxonfjvipon if this assertion fails, it won't be possible to understand what exactly was the problem: the `target` is absent or `~/.eo`? It's better to have individual `assert` statements per each particular error you are checking.

Reply:
fixed